src/components/DropArea/components/SbDropAreaUpload.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <SbBlockUi>
+    <div class="sb-drop-area__upload">
+      <SbIcon
+        name="refresh"
+        size="small"
+        color="primary"
+      />
+      <span class="sb-drop-area__upload-label">
+        {{ returnLabelToUpload }}
+      </span>
+      <span class="sb-drop-area__time-left">
+        {{ returnTimeLeft }}
+      </span>
+      <SbLoading
+        type="bar"
+        :value="percentageValue"
+      />
+    </div>
+  </SbBlockUi>
+</template>
+
+<script>
+import SbIcon from '../../Icon'
+import SbLoading from '../../Loading'
+import SbBlockUi from '../../BlockUI'
+
+export default {
+  name: 'SbDropAreaUpload',
+
+  components: {
+    SbIcon,
+    SbLoading,
+    SbBlockUi
+  },
+
+  props: {
+    totalFiles: {
+      type: [String, Number],
+      default: 0
+    },
+    actualFile: {
+      type: [String, Number],
+      default: 0
+    },
+    fileName: {
+      type: String,
+      default: null
+    },
+    percentageValue: {
+      type: [String, Number],
+      default: 0
+    },
+    timeLeft: {
+      type: [String, Number],
+      default: 0
+    }
+  },
+
+  computed: {
+    returnLabelToUpload () {
+      return `Uploading ${this.actualFile}/${this.totalFiles} - ${this.fileName}`
+    },
+    returnTimeLeft () {
+      return `${this.timeLeft} sec left`
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+.sb-drop-area__upload {
+    border-radius: 5px;
+    background-color: $white;
+    width: 100%;
+    max-width: 602px;
+    height: 49px;
+    margin: 0 auto;
+    position: relative;
+    box-shadow: 0px 3px 44px 13px rgba(0, 0, 0, 0.08);
+
+    svg {
+      float: left;
```

Comment:
You can use flex in parent component, and you don't need to use float in children. Besides that, you can use `padding` instead `margin` with `rem` unit.